### ISSUE TITLE
Match README badge to actual license, or add to metadata make dynamic

### ISSUE THREAD

**Issue by kwinkunks** [equinor/webviz-ert#410]
The license on the project is GPL, based on the [`LICENSE`](https://github.com/equinor/webviz-ert/blob/main/LICENSE) file.

However, the README contains the badge for the AGPL. Additionally, the license is not named in the project metadata in `setup.py` or anywhere else.

Since we publish the package to PyPI, if we included the licence in the metadata we could pick it up from there with, e.g. 

```
[![PyPI license](https://img.shields.io/pypi/l/webviz-ert.svg)](https://pypi.org/project/webviz-ert/)
```

Bottom line: we should at least change the badge to match the license (assuming we're keeping the GPL, which I'd recommend).
